viewing_party/party.py

```diff
@@ -1,23 +1,220 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movies_dict = {}
+    if not title or not genre or not rating:
+        return None
+    else:
+        movies_dict["title"] = title
+        movies_dict["genre"] = genre
+        movies_dict["rating"] = rating 
+    return movies_dict

```

Comment:
One way to shorten this could be like the following: 
```python
if title and genre and movie:
     return {
         "title": title,
         "genre": genre,
         "rating": rating
     }
     
return None
```